pylint/checkers/classes/class_checker.py

```diff
@@ -579,12 +579,13 @@ def _has_same_layout_slots(slots, assigned_value):
         "Used when an __init__ method is called on a class which is not "
         "in the direct ancestors for the analysed class.",
     ),
-    "W0235": (
-        "Useless super delegation in method %r",
-        "useless-super-delegation",
+    "W0246": (
+        "Useless parent delegation in method %r",
+        "useless-parent-delegation",
         "Used whenever we can detect that an overridden method is useless, "
-        "relying on super() delegation to do the same thing as another method "
+        "relying on parent delegation to do the same thing as another method "
         "from the MRO.",
+        {"old_names": [("W0235", "useless-super-delegation")]},
```

Comment:
Using a tuple here trip up the check for ``consider-using-namedtuple-or-dataclass`` for the whole dict.